Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1427,9 +1427,9 @@ PhysicalParticleContainer::SplitParticles(int lev)
         // before splitting results in a uniform distribution after splitting
         const amrex::Vector<int> ppc_nd = plasma_injector->num_particles_per_cell_each_dim;
         const std::array<Real,3>& dx = WarpX::CellSize(lev);
-        amrex::Vector<amrex::Real> split_offset = {dx[0]/2./ppc_nd[0],
-                                                   dx[1]/2./ppc_nd[1],
-                                                   dx[2]/2./ppc_nd[2]};
+        amrex::Vector<Real> split_offset = {dx[0]/2._rt/ppc_nd[0],
```

Comment:
to me more precise, this is mixing int and real operations and should explicitly cast to:
```C++
dx[0]/(2._rt * Real(ppc_nd[0]))
```